From 5e102427efe09f7f93cf35b7bf7e49b7ed721745 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 10 Feb 2022 15:43:31 +0200
Subject: [PATCH 38/43] clk: Remove inclusion of clk dt-bindings

Removed all "#include" directives of non-SCMI-related
s32* include files from include/dt-bindings/clock/.

Issue: ALB-8352
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/cpu/armv8/s32/s32-gen1/soc.c | 1 -
 arch/arm/dts/fsl-s32-gen1.dtsi        | 2 --
 arch/arm/dts/fsl-s32g.dtsi            | 1 -
 arch/arm/dts/fsl-s32g3.dtsi           | 1 -
 arch/arm/dts/fsl-s32r45.dts           | 1 -
 board/freescale/s32-gen1/eth.c        | 1 -
 drivers/clk/s32/clk_utils.c           | 1 -
 drivers/net/pfeng/pfeng_cmd.c         | 1 -
 8 files changed, 9 deletions(-)

diff --git a/arch/arm/cpu/armv8/s32/s32-gen1/soc.c b/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
index febc3797bd..f11fe07907 100644
--- a/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
+++ b/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
@@ -21,7 +21,6 @@
 #include <dm/uclass.h>
 #include <generic-phy.h>
 #endif
-#include <dt-bindings/clock/s32gen1-clock.h>
 #include <s32gen1_clk_utils.h>
 
 DECLARE_GLOBAL_DATA_PTR;
diff --git a/arch/arm/dts/fsl-s32-gen1.dtsi b/arch/arm/dts/fsl-s32-gen1.dtsi
index 39495c6d3a..bd739de431 100644
--- a/arch/arm/dts/fsl-s32-gen1.dtsi
+++ b/arch/arm/dts/fsl-s32-gen1.dtsi
@@ -6,8 +6,6 @@
  */
 
 #include <dt-bindings/interrupt-controller/irq.h>
-#include <dt-bindings/clock/s32gen1-clock-freq.h>
-#include <dt-bindings/clock/s32gen1-clock.h>
 #include <dt-bindings/clock/s32gen1-scmi-clock.h>
 
 / {
diff --git a/arch/arm/dts/fsl-s32g.dtsi b/arch/arm/dts/fsl-s32g.dtsi
index b535d95349..f361414a60 100644
--- a/arch/arm/dts/fsl-s32g.dtsi
+++ b/arch/arm/dts/fsl-s32g.dtsi
@@ -7,7 +7,6 @@
  * the Free Software Foundation; either version 2 of the License, or
  * (at your option) any later version.
  */
-#include <dt-bindings/clock/s32g-clock.h>
 #include <dt-bindings/clock/s32g-scmi-clock.h>
 #include <dt-bindings/pinctrl/s32g-pinctrl.h>
 
diff --git a/arch/arm/dts/fsl-s32g3.dtsi b/arch/arm/dts/fsl-s32g3.dtsi
index 0d6394e325..409f04db8f 100644
--- a/arch/arm/dts/fsl-s32g3.dtsi
+++ b/arch/arm/dts/fsl-s32g3.dtsi
@@ -7,7 +7,6 @@
  * the Free Software Foundation; either version 2 of the License, or
  * (at your option) any later version.
  */
-#include <dt-bindings/clock/s32g3-clock.h>
 
 /delete-node/ &gic;
 / {
diff --git a/arch/arm/dts/fsl-s32r45.dts b/arch/arm/dts/fsl-s32r45.dts
index c2ae007ec4..b5366a8101 100644
--- a/arch/arm/dts/fsl-s32r45.dts
+++ b/arch/arm/dts/fsl-s32r45.dts
@@ -7,7 +7,6 @@
  * the Free Software Foundation; either version 2 of the License, or
  * (at your option) any later version.
  */
-#include <dt-bindings/clock/s32r45-clock.h>
 #include <dt-bindings/clock/s32r45-scmi-clock.h>
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/pinctrl/s32r45-pinctrl.h>
diff --git a/board/freescale/s32-gen1/eth.c b/board/freescale/s32-gen1/eth.c
index 59d8e5cd77..288edf07a0 100644
--- a/board/freescale/s32-gen1/eth.c
+++ b/board/freescale/s32-gen1/eth.c
@@ -21,7 +21,6 @@
 #endif
 #include <fdt_support.h>
 #include <clk.h>
-#include <dt-bindings/clock/s32gen1-clock.h>
 #include <s32gen1_clk_utils.h>
 #include <s32gen1_gmac_utils.h>
 #include <dm/device_compat.h>
diff --git a/drivers/clk/s32/clk_utils.c b/drivers/clk/s32/clk_utils.c
index 19f32aaf1d..003473dc38 100644
--- a/drivers/clk/s32/clk_utils.c
+++ b/drivers/clk/s32/clk_utils.c
@@ -5,7 +5,6 @@
 
 #include <asm/arch/clock.h>
 #include <dm/uclass.h>
-#include <dt-bindings/clock/s32gen1-clock.h>
 #include "s32gen1_clk_utils.h"
 #include <s32gen1_clk_funcs.h>
 
diff --git a/drivers/net/pfeng/pfeng_cmd.c b/drivers/net/pfeng/pfeng_cmd.c
index e8f70dd2fb..93f30717b3 100644
--- a/drivers/net/pfeng/pfeng_cmd.c
+++ b/drivers/net/pfeng/pfeng_cmd.c
@@ -19,7 +19,6 @@
 #include <asm/arch/s32-gen1/serdes_xpcs_regs.h>
 #include <linux/string.h>
 #include <clk.h>
-#include <dt-bindings/clock/s32g-clock.h>
 #include <s32gen1_clk_utils.h>
 #include <dm/device_compat.h>
 #include <dm/pinctrl.h>
-- 
2.17.1

